ui/app/src/views/dashboard/Panel.tsx

```diff
@@ -29,6 +29,8 @@ export interface PanelProps extends CardProps {
 function Panel(props: PanelProps) {
   const { definition, ...others } = props;
   const [contentElement, setContentElement] = useState<HTMLDivElement | null>(null);
+  const isStatsChart = definition.kind === 'StatChart';
```

Comment:
This is nitpicky, but should this be isStatChart? Or we could potentially rename the component / panel definition to StatsChart too